components/kubeflow/katib-launcher/component.yaml

```diff
@@ -1,40 +1,38 @@
+- {name: Experiment name,           type: String,                       description: 'Experiment name.'}
 - {name: Namespace,                 type: String,   default: kubeflow,  description: 'Namespace.'}
-- {name: Optimization type,         type: String,   default: minimize,  description: 'Direction of optimization. minimize or maximize.'}
-- {name: Objective value name,      type: String,                       description: 'Objective value name which trainer optimizes.'}
-- {name: Optimization goal,         type: Float,                        description: 'Stop studying once objectivevaluename value exceeds optimizationgoal.'}
-- {name: Request count,             type: Integer,  default: 1,         description: 'Number of requests to the suggestion service.'}
-- {name: Metrics names,             type: String,                       description: 'List of metric names (comma-delimited).'}
-- {name: Parameter configs,         type: YAML,     default: '',        description: 'Parameter configs (YAML/JSON format).'}
-- {name: NAS config,                type: YAML,     default: '',        description: 'NAS config (YAML/JSON format).'}
-- {name: Worker template path,      type: String,   default: '',        description: 'Worker spec.'}
-- {name: Metrics collector template path, type: String, default: '',    description: 'Metrics collector spec.'}
-- {name: Suggestion spec,           type: YAML,     default: '',        description: 'Suggestion spec (YAML/JSON format).'}
-- {name: StudyJob timeout minutes,  type: Integer,  default: '10',      description: 'Time in minutes to wait for the StudyJob to complete.'}
-- {name: Delete finished job,       type: Bool,  default: 'True',    description: 'Whether to delete the job after it is finished.'}
+- {name: Version,                   type: String,   default: v1alpha3,  description: 'Experiment version.'}
+- {name: Max trial count,           type: Integer,                      description: 'How many trials can be executed at most.'}
+- {name: Max failed trial count,    type: Integer,                      description: 'How many trials can fail at most.'}
+- {name: Parallel trial count,      type: Integer,  default: 3          description: 'How many trials can be running in parallel at most.'}
+- {name: Objective Configuration,   type: JSON,                         description: 'Experiment objective.'}
+- {name: Algorithm Configuration,   type: JSON,                         description: 'Experiment algorithm.'}
+- {name: trialTemplate,             type: JSON,                         description: 'Experiment trialTemplate.'}
+- {name: Parameter configs,         type: JSON,                         description: 'Experiment Parameter configuration.'}
+- {name: Metrics collector,         type: JSON,                         description: 'Experiment metricsCollector.'}
+- {name: Experiment timeout minutes, type: Integer,  default: 60*24,    description: 'Time in minutes to wait for the Experiment to complete.'}
```

Comment:
>60*24

I'm not sure this will work as it's not an integer.

Reply:
it can work, in fact it is 1 day (1440 min)

Reply:
to avoid confused, change it to 1440

Reply:
`components/kubeflow/katib-launcher/component.yaml` is generated by me manually.
what is this file used for? if there is any tool to generate it @Ark-kun

Reply:
> it can work, in fact it is 1 day (1440 min)

I'm curious. Can you tell be why/how this works?

Reply:
@Ark-kun sorry to mislead you, when I replied you, I didn't know what component.yaml is used for, I had thought you were asking me if `experimentTimeoutMinites` works.
Now, I had fixed it and test `component.yaml ` in sample2.py
Thanks